learn/framework/templates.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Templates
+description: Substrate is a powerful blockchain development framework that provides many ready-to-use templates for building parachains, nodes and more, and allowing product teams to quickly and easily build Appchains.
+---
+
+# Templates {: #templates } 
```

Comment:
As discussed, we should probably expand this and put it inside the Tanssi section?

Reply:
Leaving the sections out (hidden) for the moment, to expand and publish in another PR